File: src/JavaScript/Sibice/Sibice.js
const fs = require('fs');
const readinput = fs.readFileSync(0, 'utf8').split('\n');
let current = 0;

function input() {
    return readinput[current++];
}

[n, w, h] = input().split(' ');
let hyp = Math.sqrt(w * w + h * h);
while (n-- > 0) {
    if (input() <= hyp) {
        console.log('DA');
    } else {
        console.log('NE');
    }
}

File: src/Java/Sibice/Sibice.java
public class Sibice {
    public static void main(String args[]) {
        Kattio io = new Kattio(System.in);
        int n = io.getInt(), w = io.getInt(), h = io.getInt();
        double py = Math.sqrt(w * w + h * h);
        for (int i = 0; i < n; i++) {
            io.println(io.getInt() <= py ? "DA" : "NE");
            io.flush();
        }
    }
}